junipersrx.json

```diff
@@ -557,51 +557,51 @@
             },
             {
                 "module": "lxml",
-                "input_file": "wheels/py36/lxml-5.3.0-cp36-cp36m-manylinux_2_28_x86_64.whl"
+                "input_file": "wheels/py39/lxml-5.3.0-cp39-cp39-manylinux_2_28_x86_64.whl"
             },
             {
                 "module": "ncclient",
                 "input_file": "wheels/shared/ncclient-0.6.12-py2.py3-none-any.whl"
             },
             {
                 "module": "paramiko",
-                "input_file": "wheels/py3/paramiko-3.3.1-py3-none-any.whl"
+                "input_file": "wheels/py3/paramiko-3.5.0-py3-none-any.whl"
             },
             {
                 "module": "ply",
                 "input_file": "wheels/shared/ply-3.11-py2.py3-none-any.whl"
             },
             {
                 "module": "pycparser",
-                "input_file": "wheels/shared/pycparser-2.21-py2.py3-none-any.whl"
+                "input_file": "wheels/py3/pycparser-2.22-py3-none-any.whl"
             },
             {
                 "module": "setuptools",
-                "input_file": "wheels/py3/setuptools-59.6.0-py3-none-any.whl"
+                "input_file": "wheels/py3/setuptools-75.3.0-py3-none-any.whl"
             },
             {
                 "module": "simplejson",
-                "input_file": "wheels/py36/simplejson-3.17.2-cp36-cp36m-manylinux2010_x86_64.manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_28_x86_64.whl"
+                "input_file": "wheels/py39/simplejson-3.17.2-cp39-cp39-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_28_x86_64.whl"
             }
         ]
     },
-    "pip39_dependencies": {
+    "pip_dependencies": {
```

Comment:
Please delete all the wheels and all `*_dependency" keys from json and rerun dependencies generation as it looks like some stale packages remain + maybe pip_dependencies won't be regenerated at all.

Reply:
Doing this results in packages we don't want being added, such as setuptools <70.0.0. I think there are changes need to be made to the pre-commit, specifically the package app dependencies step, in order to fix this problem.